.travis.yml

```diff
@@ -1,20 +1,21 @@
 ---
 language: node_js
 node_js:
-  - "5"
   - "4"
-  - "0.12"
+  - "6"
+  - "stable"
```

Comment:
Mild preference for calling this `7` since https://github.com/creationix/nvm/blob/f1a5f2a15fd2867a4a5762b80cae028d324ee6a1/README.markdown#L182